tests/phpunit/elementor/modules/atomic-widgets/styles/test-styles-renderer.php

```diff
@@ -390,7 +390,8 @@ public function test_render__style_with_nested_background_transformers() {
 														],
 													],
 													'position' => 'top center',
-													'resolution' => 'medium'
+													'resolution' => 'medium',
```

Comment:
We will need to update the snapshot files as well. 

See e.g. https://github.com/elementor/elementor/blob/4390e6457834a1aca360fb5522e48eda960b2678/tests/phpunit/elementor/modules/atomic-widgets/styles/__snapshots__/Test_Styles_Renderer__test_render__style_with_nested_background_transformers__1.txt#L1